.changelog/4427.internal.md

```diff
@@ -0,0 +1,14 @@
+go: bump tendermint to 0.35
+
+Configuration changes required due to Tendermint changes:
```

Comment:
Can you split the described configuration changes to a separate `4427.cfg.md` Change Log fragment so it will be properly grouped together under "Configuration Changes" section of the Change Log?